code/commands/PlayerCommands.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.Linq;
+using Sandbox;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.Commands
+{
+public static class PlayerCommands
+{
```

Comment:
i'm still not sure how i feel about the non-indentation for the namespace :/ i get the 'unneeded indentation' point but it still looks exceedingly weird